src/app/components/navbar.jsx

```diff
@@ -4,6 +4,49 @@ import Search from "./search";
 import Link from "next/link";
 import Switch from "@mui/material/Switch";
 
+const SunIcon = ({ className = "" }) => (
```

Comment:
Can you create a file called icons in the components and just import it from there?